run_reasoner.sh

```diff
@@ -151,7 +151,7 @@ echo "Launching: mros reasoner"
 gnome-terminal --window --geometry=80x24+10+10 -- bash -c "source $PYTHON3_VENV_PATH/venv3.6_ros/bin/activate;
 source $PYTHON3_VENV_PATH/devel/setup.bash;
 source $REASONER_WS_PATH/devel/setup.bash;
-roslaunch mros1_reasoner run.launch onto:=kb.owl;
+roslaunch mros1_reasoner run.launch onto:=$REASONER_WS_PATH/src/mros1_reasoner/mros1_reasoner/scripts/kb.owl;
```

Comment:
Why not use `$(rospack find <your_pkg_name>)`?

Reply:
`rospack` is a regular Python script, not a `roslaunch` substitution command.

Reply:
I should follow Robotic Software Practicals...  :-)